subprojects/settings.gradle.kts

```diff
@@ -299,7 +299,7 @@ dependencyResolutionManagement {
 }
 
 plugins {
-    id("com.gradle.enterprise") version "3.5.1"
+    id("com.gradle.enterprise") version "3.6.1"
```

Comment:
- Reduced memory usage for when capturing task file inputs in very large builds
- Avoid occasional build scan capture errors during IDE sync
- The `GRADLE_ENTERPRISE_ACCESS_KEY` environment variable allows specifying the access keys for multiple hosts
- Reduced memory usage for builds with lots of input files